Makefile

```diff
@@ -0,0 +1,27 @@
+REGISTRY_NAME := 
+REPOSITORY_NAME := sindastra/
+IMAGE_NAME := spigotmc-build
+TAG := :latest
+
+.PHONY: getcommitid
+
+
+
+all: build
+
+getcommitid: 
+	$(eval COMMITID = $(shell git log -1 --pretty=format:"%H"))
+
+build: getcommitid
+	docker build -t $(REGISTRY_NAME)$(REPOSITORY_NAME)$(IMAGE_NAME)$(TAG) hub/.
+	docker tag $(REGISTRY_NAME)$(REPOSITORY_NAME)$(IMAGE_NAME)$(TAG) $(REGISTRY_NAME)$(REPOSITORY_NAME)$(IMAGE_NAME):$(COMMITID)
+
+run:
+	docker run -d $(RUN_PORTS) $(REGISTRY_NAME)$(REPOSITORY_NAME)$(IMAGE_NAME)$(TAG)

```

Comment:
What is ```$(RUN_PORTS)```?

Reply:
That is a holdover from the vscode snippet that I use to boilerplate my makefiles. most of my docker work is web services. I removed it/replaced it with the --name example from the readme.